### ISSUE TITLE
1D Maxwellian example update

### ISSUE THREAD

**Issue by lemmatum** [PlasmaPy/PlasmaPy#427]
The 1D Maxwellian [example](http://docs.plasmapy.org/en/stable/auto_examples/plot_distribution.html#sphx-glr-auto-examples-plot-distribution-py) in our docs should have a plot for 1D speed distribution with lines that demarcate the various thermal velocities (i.e. "most probable", "mean magnitude", etc.).

**Comment by lemmatum** [comment]
Here is an example

![image](https://user-images.githubusercontent.com/28945309/39843137-f91a4572-53a6-11e8-9fc8-ce30d426a46c.png)
